### ISSUE TITLE
[Introduction to Ensembling/Stacking in Python] TIL

### ISSUE THREAD

**Issue by dduniverse** [dduniverse/kaggle-copy#3]
from sklearn.**cross_validation** import KFold
↓
from sklearn.**model_selection** import KFold

**Comment by dduniverse** [comment]
**np.random.randint(m, n, size)**
- m부터 n까지의 수 중 난수 1개 생성
- size: 난수 개수
- size=(2,3)이면 (2,3) 형태의 array로 생성

**Comment by dduniverse** [comment]
**pd.cut**: 모든 구간을 같은 길이로 나누기
- 각 구간의 길이 4 → (1,5] (5, 9] ...
**pd.qcut**: 모든 구간을 같은 개수로 나누기
- 각 구간의 개수 4 → (1,3,4,6) (7,9,10,13) ...

**Comment by dduniverse** [comment]
**u'Age**'

- 문자열 Age 유니코드 문자열임을 의미
- Python 3.x에서 문자열은 기본적으로 유니코드를 사용하므로 u접두사 가 필요하지 않음

**Comment by dduniverse** [comment]
**KFold(n_splits=2, random_state=None, shuffle=False)**

- n_splits : split 개수
- random_state: 시드
- shuffle: T/F

Changed in version 0.22: **n_splits** default value changed from 3 to 5.
n_folds → n_splits

**Comment by dduniverse** [comment]
**np.ravel()**
numpy 배열을 1차원으로 평평하게 만들어주는 함수
